eternalcore-core/src/main/java/com/eternalcode/core/bridge/litecommand/argument/PlayerArgument.java

```diff
@@ -28,7 +29,10 @@ public PlayerArgument(ViewerService viewerService, TranslationManager translatio
 
     @Override
     public ParseResult<Player> parse(Invocation<CommandSender> invocation, String argument, Translation translation) {
-        Player target = this.server.getPlayer(argument);
+        Player target = this.server.getOnlinePlayers().stream()
+            .filter(player -> player.getName().equalsIgnoreCase(argument))
+            .findFirst()
+            .orElse(null);
```

Comment:
This solution is not very efficient. Imagine that this function will be executed 30 times with 1000 people on the server in same time. Use the #getPlayerExact method which searches case insensitively. 


```suggestion
        Player target = this.server.getPlayerExact(argument);
```

Reply:
⚡